Packs/GoogleApigee/Integrations/GoogleApigeeEventCollector/GoogleApigeeEventCollector.py

```diff
@@ -116,9 +109,32 @@ def get_token_request(self, refresh_token: str = '') -> tuple[str, str, str]:
         }
         zone = f'{self.zone}.' if self.zone else ''
         url = f'https://{zone}login.apigee.com/oauth/token'
-        token_response = self._http_request('POST', full_url=url, url_suffix='/oauth/token', data=data, headers=headers)
+        try:
+            token_response = self._http_request('POST', full_url=url, url_suffix='/oauth/token', data=data, headers=headers)
+        except Exception as e:
```

Comment:
Please add. debug log here explaining what happened and how we handled the case. 
Include also the original error that was caught.